frontend/components/Slider/index.jsx

```diff
@@ -38,7 +44,16 @@ const Slider = ({ products, type, settings }) => {
   return (
     <div className={wrapper}>
       <Header {...headerProps} />
-      <ProductSlider productIds={productIds} />
+      <ProductSlider
+        productIds={productIds}
+        meta={{
+          isRecommendation: true,
+          recommendationScope: {
+            id,
```

Comment:
`type` is still missing in the `recommendationScope` object. it's needed for the selectors inside the dynamic yield extension to work.